.pre-commit-config.yaml

```
@@ -44,7 +44,7 @@ repos:
   hooks:
     - id: codespell
       args:
-        - --ignore-words-list=degreeE,degreee,varn,poit,uint,sur,herat
+        - --ignore-words-list=degreeE,degreee,varn,poit,uint,sur,herat,Claus
```

Checked it locally and it does work:

```suggestion
        - --ignore-words-list=degreeE,degreee,varn,poit,uint,sur,herat,claus
```

---

BTW, the `degreeE` there is probably not needed.